packages/toolpad-core/babel.config.js

```diff
@@ -0,0 +1,144 @@
+const path = require('path');
+
+const errorCodesPath = path.resolve(__dirname, './docs/public/static/error-codes.json');
+const missingError = process.env.MUI_EXTRACT_ERROR_CODES === 'true' ? 'write' : 'annotate';
+
+function resolveAliasPath(relativeToBabelConf) {
+  const resolvedPath = path.relative(process.cwd(), path.resolve(__dirname, relativeToBabelConf));
+  return `./${resolvedPath.replace('\\', '/')}`;
+}
+
+const productionPlugins = [
+  ['babel-plugin-react-remove-properties', { properties: ['data-mui-test'] }],
+];
+
+module.exports = function getBabelConfig(api) {
+  const useESModules = api.env(['regressions', 'stable', 'rollup']);
+
+  const defaultAlias = {
+    '@toolpad/core': resolveAliasPath('./packages/toolpad-core/src'),
```

Comment:
No aliasing in babel. it's not the right tool for that. I see we're using `vitest` for testing, if you need aliasing, do it in `vitest`.

You can also remove the error codes macro. We'll add this optimization later if we need it. We're going to figure out a more portable method to support it.

In fact `babel.config.js` is only for top-level configuration. If possible, we should remove this file altogether.